alma/lib/Validators/PaymentValidation.php

```diff
@@ -268,6 +274,10 @@ public function validatePayment($almaPaymentId)
                 }
             }
 
+            $planKey = $this->settingsHelper->key('general', $payment->installments_count, $payment->deferred_days, $payment->deferred_months);
```

Comment:
use static function keyPlanStatic...

Reply:
I created a new static function because the function keyForInstallmentPlanStatic is compatible for Eligibility Plan only